rllib/core/models/catalog.py

```diff
@@ -224,3 +240,80 @@ def get_tokenizer_config(
                 **{"use_lstm": False, "use_attention": False},
             },
         )
+
+    @classmethod
+    def get_action_dist_cls_dict(
+        cls,
+        action_space: gym.Space,
+        model_config_dict: ModelConfigDict,
+        deterministic: Optional[bool] = False,
+    ) -> Mapping[str, Any]:
+        """Returns a mapping from framework to distribution class.
+
+        You can get the required input dimension for the distribution by calling
+        `action_dict_cls.required_model_output_shape(action_space, model_config_dict)`
+        on the retrieved class. This is useful, because the Catalog needs to find out
+        about the required input dimension for the distribution before the model that
+        outputs these inputs is configured.
+
+        Args:
+            action_space: Action space of the target gym env.
+            model_config_dict: Optional model config.
+            deterministic: Whether to return a Deterministic distribution instead of
```

Comment:
better wording: "Whether to return a Deterministic distribution on input logits instead of a stochastic distributions. For example for Discrete spaces, the stochastic is a Categorical distribution with output logits, while the deterministic distribution will be to output the argmax of logits directly."

Reply:
Thanks putting in the effort of correcting these in my PRs! 🙂